src/test/test-honor-first-shutdown.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Honor First Shutdown feature
+After=multi-user.target
+
+[Service]
+ExecStart=/home/test-honor-first-shutdown.sh
+ExecStop=pkill -9 test.sh
+FailureAction=reboot
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
This test is not hooked up anywhere that I can see... Is it supposed to be doing something?

Reply:
Created a new test/TEST-47-HONORFIRSTSHUTDOWN/ which will be seen when we resolve the question below.